public/javascripts/CreateUser.js

```diff
@@ -7,12 +7,8 @@ jQuery('#createUser').submit(function(){
         "lastName":jQuery('#last').val()
     };
     //console.log(toPass);
-    jQuery.ajax({
-        type: "POST",
-        url: window.apiRoute+'/login/createUser/',
-        data: toPass,
-        dataType: 'json',
-        success: function(){
+    navigation.postJSON(window.apiRoute+'/login/createUser/', toPass, function(err, res){
+        if(res) {
```

Comment:
There was no case for that before, I will add one with John's suggested wording now.